### ISSUE TITLE
Feature request: set starting slide index

### ISSUE THREAD

**Issue by MacPrawn** [SSENSE/vue-carousel#77]
Great slider - thank you very much!

I was wondering if we could be allowed to set the first slide index? I would like to start my slider on the very last slide in the list. Or maybe there's already a way to do this that I have missed?

Thanks!

**Comment by christan0612** [comment]
I have same issue
and this is my solution

give carousel a reference
<carousel ref="carousel">...</carousel>
and set:
this.$refs.carousel.currentPage = yourStartIndex

if anyone have better solution please let me know
thanks

**Comment by daniwp** [comment]
@quinnlangille I don't believe this works. I have tried doing what you suggested but without success.
